project/urbanity/urbanity.vcxproj.filters

```diff
@@ -37,6 +37,12 @@
     <Filter Include="Source Files\simulator">
       <UniqueIdentifier>{d002ebc7-fb60-4767-ad16-d81ad52c6b8f}</UniqueIdentifier>
     </Filter>
+    <Filter Include="Header Files\Piranha">
```

Comment:
All filters currently follow a lowercase only naming convention so we should consider being consistent about it